### ISSUE TITLE
outstanding issues

### ISSUE THREAD

**Issue by mikekestemont** [emanjavacas/bntl#30]
- [x] a side widget for showing the latest posts on X (https://x.com/_BNTL_) and Bluesky
- [x] a mailto-button (carl.destrycker@kbr.be), so that people can signal missing publications
- [ ] different icons per publication type (Carl/Eva will provide these)
- [x] publications should get a field indicating whether a UR field (with a url) is present, which indicates that they are open access.
       - [ ] if publications are OA (see previous bullet), they should get an additional icon (an open hanglock)
       - [ ] possibility to filter search search results on whether publications are OA or not
- [ ] more options (Something human-readable, instead of structured, e.g. MLA) to export individual publications or groups of them. A list of options would be nice.
- [x] The export function is a bit hidden now: could you insert an export/save icon instead of an arrow to make this more visible?
- [x] A help-page (can be a stub now, content will be provided by Carl/Eva, since they want to embed a YouTube video)
- [ ] Google analytics integration (Carl/eva will provide a gmail address for linking it to)
- [x] Use the new logo (attached)

[BNTL logo_Kleur (CMYK).pdf](https://github.com/user-attachments/files/18151773/BNTL.logo_Kleur.CMYK.pdf)

**Comment by emanjavacas** [comment]
the SV tag, which appears in your docs as Series number (for CHAP) isn't recognised by rispy (according to wikipedia it corresponds to series volume).

This tag doesn't appear in any of the documents you've provided.

Should I try to patch the ris parser to accomodate this or just drop it?

**Comment by emanjavacas** [comment]
for the rest, I wait until I receive the gmail address and the icons.

**Comment by evadecooman** [comment]
Hi Enrique, 

Our gmail adress is bntl.bibliografie@gmail.com. As for the icons, they are being designed by the same person who did the logo. We're still waiting for that. 

Best wishes, Eva

**Comment by evadecooman** [comment]
About the SV tag, this cannot be dropped. It is an important part of information.

**Comment by evadecooman** [comment]
About the conversion to MLA, APA...: this is a common thing used by websites (see example below):

![image](https://github.com/user-attachments/assets/0a0a5672-247e-419f-8dd8-eb0ae2c94e55)

**Comment by mikekestemont** [comment]
About the SV-tag: I assume this is an oversight in my parsing code. Eva,
can you provide 2-3 examples in the RDF-XML where this should come through?

**Comment by evadecooman** [comment]
Yes, of course! Could you send me the last parsed-file from december? Just to be sure i'm looking in the right document.

**Comment by evadecooman** [comment]
[examples SV.zip](https://github.com/user-attachments/files/18317234/examples.SV.zip)

These are three examples exported from Zotero that have a series number (here all books)

**Comment by mikekestemont** [comment]
About the SV tag, good catch: right now this is written to 'note' <-> 'M1' (which is wrong). I would suggest to patch the rispy writer, since it *is* a valid field in the RIS specification. We can use the value "series_volume" in these lines (instead of 'note'):
- https://github.com/emanjavacas/bntl/blob/c22e21c08b20acd7a8745466caab3ff2be636bd2/bntl/rdf.py#L276
- https://github.com/emanjavacas/bntl/blob/c22e21c08b20acd7a8745466caab3ff2be636bd2/bntl/rdf.py#L520
- https://github.com/emanjavacas/bntl/blob/c22e21c08b20acd7a8745466caab3ff2be636bd2/bntl/rdf.py#L637

Then, we add "SV" <-> "series_volume"  to the default TAG_KEY_MAPPING for the writer. Would that work, @emanjavacas ?

**Comment by emanjavacas** [comment]
Alright, I've patched it already and updated the rdf.py file with Mike's suggestions.

Btw. is rdf.py something that should be incorporated into the workflow, or should I expect ris file as inputs anyway?

**Comment by mikekestemont** [comment]
Great, thanks. rdf.py should be run on the server, as Eva and Carl can't run it independently of me.

**Comment by emanjavacas** [comment]
Closing in favor of #38 #37 and #39
